guides/common/modules/proc_configuring-satellite-for-outgoing-emails.adoc

```diff
@@ -3,11 +3,11 @@
 [id="Configuring_Server_for_Outgoing_Emails_{context}"]
 = Configuring {ProjectServer} for outgoing emails
 
-To send email messages from {ProjectServer}, you can use either an SMTP server, or the `sendmail` command.
+To send email messages from {ProjectServer}, you can use an SMTP server or the deprecated `sendmail` mail transfer agent (MTA).
 
 .Prerequisites
-* Some SMTP servers with anti-spam protection or grey-listing features are known to cause problems.
-To setup outgoing email with such a service either install and configure a vanilla SMTP service on {ProjectServer} for relay or use the `sendmail` command instead.
+* Some SMTP servers with anti-spam protection or gray-listing features are known to cause problems.
+To set up outgoing email with such a service, install and configure an uncustomized SMTP service on {ProjectServer} for relay. Alternatively, you can use the deprecated `sendmail` command
```

Comment:
I'd leave out `uncustomized` because I have no idea what it would mean. Even `vanilla` is better IMHO because there I immediately have some idea that it's supposed to be just plain SMTP.

Also, 1 sentence per line.
```suggestion
To set up outgoing email with such a service, install and configure an SMTP service on {ProjectServer} for relay.
Alternatively, you can use the deprecated `sendmail` command.
```